Felienne/hedy
# [BUG FIX] Makes sure submit button is directly aligned correctly on share

**Description**
In this PR we fix a very small bug (but it has annoyed me for quite some time). When visiting the programs page and sharing a non-shared program the UI is automatically updated without a re-load, hiding the share button and showing an unshare, copy link and submit button. However, for some reason the alignment of the submit button was incorrect when doing this dynamically. When simply re-loading it worked fine. This resulted in an inconsistent view until a re-load, as can be seen on the screenshot of the related issue. Even weirder: this is due to using Tailwind! When using the `hidden` class this error occurs, however when using `style="display: none;"` it does not. Even when the class code is identical.

**Fixes**
This PR fixed #2562.

**How to test**
Make sure you have a shared and a non-shared program and navigate to the programs page. When sharing the non-shared program verify that the submit button is directly aligned correctly to the right-side of the program box.
